src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -75,14 +79,25 @@ class ZHAConfigDashboard extends LitElement {
 
   @state() private _networkSettings?: ZHANetworkSettings;
 
+  @state() private _devices: ZHADevice[] = [];
+
   @state() private _generatingBackup = false;
 
-  protected firstUpdated(changedProperties: PropertyValues): void {
+  @state() private _statusIcon = mdiCheckCircle;
+
+  @state() private _statusClass = "connected";
+
+  @state() private _total = 0;
+
+  @state() private _offline = 0;
```

Comment:
You should combine `_fetchDevices` and `_updateStatus`.

Remove states: `_devices`, `_statusIcon`, `_statusClass`

Rename `_total` to `_totalDevices`, and `_offline` to `_offlineDevices`

And do the `_statusIcon`, `_statusClass` inside the `render` func.